packages/maskbook/src/components/InjectedComponents/ToolboxHint.tsx

```diff
@@ -223,6 +227,10 @@ export function ToolboxHint(props: ToolboxHintProps) {
                 <Image src={ToolIconURLs.claim.image} width={19} height={19} />
                 <Typography className={classes.text}>{ToolIconURLs.claim.text}</Typography>
             </MenuItem>,
+            <MenuItem onClick={openCreateImportDialog} className={classes.menuItem}>
+                <Image src={ToolIconURLs.claim.image} width={19} height={19} />
+                <Typography className={classes.text}>create and import</Typography>
```

Comment:
`useI18N()`

Reply:
This is used to be a temporary entry. Removed in https://github.com/DimensionDev/Maskbook/pull/3298/commits/ceb17bf35a934b93a06b2c82020d6bb7e973e963.